src/main/include/log4cxx/net/telnetappender.h

```diff
@@ -116,6 +116,16 @@ class LOG4CXX_EXPORT TelnetAppender : public AppenderSkeleton
 		*/
 		void setPort(int newValue);
 
+		/**
+		The <b>Hostname</b> on which to accept connections.
+		*/
+		LogString getHostname() const;
```

Comment:
Would 'NodeName' be bettter than 'HostName' given it identifies the local machine interface to be used? 
I believe that would correspond better with [the underlying terminology](https://man7.org/linux/man-pages/man3/getaddrinfo.3.html) and distinguish it from a url used to identify a remote server.